lib/3scale_toolbox/commands/copy_command/copy_service.rb

```diff
@@ -0,0 +1,214 @@
+require 'cri'
+require '3scale_toolbox/base_command'
+
+module ThreeScaleToolbox
+  module Commands
+    module CopyCommand
+      module CopyServiceSubcommand
+        extend ThreeScaleToolbox::Command
+        def self.command
+          Cri::Command.define do
+            name        'service'
+            usage       'service [options] <service_id>'
+            summary     '3scale CLI copy service'
+            description '3scale CLI copy service tool'
+
+            flag :h, :help, 'show help for this command' do |_, cmd|
+              puts cmd.help
+              exit 0
+            end
+
+            required  :s, :source, 'Source'
+            required  :d, :destination, 'Destination'
+            required  :t, 'target_system_name', 'Target system name'
+
+            run do |opts, args, _|
+              CopyServiceSubcommand.run opts, args
+            end
+          end
+        end
+
+        def self.run(opts, args)
+          source      = fetch_required_option(opts, :source)
+          destination = fetch_required_option(opts, :destination)
+          system_name = fetch_required_option(opts, :target_system_name)
+          exit_with_message 'error: missing service_id argument' if args.empty?
+          service_id = args[0]
+          copy_service(service_id, source, destination, system_name)
+        end
+
+        def self.exit_with_message(message)
+          puts message
+          exit 1
```

Comment:
We will need to find some abstraction for this later, so it does not exit running test suite.